src/core/core.controller.js

```diff
@@ -334,9 +334,17 @@ module.exports = function(Chart) {
 			this.tooltip.initialize();
 		},
 
-		update: function(animationDuration, lazy) {
+		update: function(config) {
 			var me = this;
 
+			if (!config || typeof config !== 'object') {
```

Comment:
We should have a test, if possible, that verifies that backwards compatibility isn't broken.